README.md

```diff
@@ -52,13 +53,12 @@ Re.Pack uses Webpack 5 and React Native CLI's plugin system to allow you to bund
   - `GET /api/:platform/assets` - assets emitted from compilation.
   - `GET /api/:platform/stats` - data about compilation.
 - [x] [Module Federation](https://medium.com/swlh/webpack-5-module-federation-a-game-changer-to-javascript-architecture-bcdd30e02669) support
+- [x] Code signing and verification
```

Comment:
I saw this was merged but not sure if you want it to be in the official feature list